src/Common/utils.ts

```diff
@@ -1,5 +1,38 @@
+import { Utils } from 'xpring-common-js/build/src'
+
 const isNode = (): boolean => {
   return process?.release?.name === 'node'
 }
 
+/**
+ * Converts a string such that each of its characters are represented as hex.
+ *
+ * @param value - the string to convert to hex.
+ * @returns the hex encoded version of the string.
+ */
+export const stringToHex = (value: string): string => {
+  let result = ''
+  // eslint-disable-next-line no-plusplus
+  for (let i = 0; i < value.length; i++) {
+    result += value.charCodeAt(i).toString(16)
+  }
+  return result
+}
+
+/**
+ * Converts a string that is optionally in hex format into a Uint8Array.
+ *
+ * @param value - the string to convert to a Uint8Array.
+ * @param isHex - flag to indicate it's a hex string or not.
+ * @returns the Uint8Array value.
+ */
+export const stringToUint8Array = (
```

Comment:
There's already a method called toBytes() in `Utils`? 

Maybe just `asciiToBytes` and you can drop the `isHex` param?

Reply:
Maybe this is just a point of confusion for me, but the fact that the memo only allows for hex strings was something that I thought made the case for more flexibility in the API since it's not obvious if the string that was passed in is already a hex string. Maybe I'm just being dumb though.

Reply:
SGTM